documentation/modules/ref-resources-table-messaging-tenant.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// master-kubernetes.adoc
+// master-openshift.adoc
+// Using master.adoc
+
+[id='ref-resources-table-messaging-tenant-{context}']
+= {ProductName} resources for messaging tenants
+
+The following table describes the {ProductName} resources that pertain to the messaging tenant role.
+
+.{ProductName} messaging tenant resources table
+[cols="50%a,50%a",options="header",subs="attributes"]
+|===
+|Resource |Description
+|`addresses`|Specifies the address.
+|`addressspaces`|Specifies the address space.
+|`addressspaceschemas`|Specifies .
```

Comment:
@k-wall, Can you please help with a proper description of `addressspaceschemas`?

Reply:
an `addressspaceschema` defines the service characteristics available to an addresspace.  An addressspace refers one addressspaceschema.  'standard' and 'brokered' are predefined addressspaceschemas.

Reply:
Fixed both issues